x-pack/platform/plugins/shared/event_log/server/types.ts

```diff
@@ -47,7 +51,7 @@ export interface IEventLogService {
 }
 
 export interface IEventLogClientService {
-  getClient(request: KibanaRequest): IEventLogClient;
+  getClient(request: KibanaRequest | string): IEventLogClient;
```

Comment:
Is this intentional?  Seems like it should cause a type error in the implementation: https://github.com/nkhristinin/kibana/blob/34ff766b635148d19a9b21f782cea217632b78ac/x-pack/platform/plugins/shared/event_log/server/event_log_start_service.ts#L41-L47

I'm not sure what it would be used for in any case ...